addons/website_slides/controllers/main.py

```diff
@@ -934,15 +959,19 @@ def slide_view(self, slide, **kwargs):
         if slide.can_self_mark_completed and not slide.user_has_completed \
            and slide.channel_id.channel_type == 'training':
             self._slide_mark_completed(slide)
+            next_category_to_open = slide.get_next_category_id() if slide.is_category_completed() else False
```

Comment:
can we move the check of `is_category_completed` inside `get_next_category_id` ?